packages/eth2-types/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@chainsafe/eth2-types",
+  "version": "0.1.0",
+  "description": "> TODO: description",
+  "author": "ChainSafe Systems",
+  "license": "GPL-3.0",
+  "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "build": "babel src -x .ts -d lib --source-maps",
+    "check-types": "tsc --noEmit --incremental",
+    "test": "nyc --cache-dir .nyc_output/.cache -r lcov -e .ts mocha -r ts-node/register 'test/**/*.test.ts' && nyc report"
```

Comment:
name this `test:unit` so we can execute unit tests in all directories with single command

Reply:
oh interesting so i can do `lerna run test:unit` instead of running the scoped version?